api/src/main/java/com/github/eyefloaters/console/api/model/ReplicaInfo.java

```diff
@@ -0,0 +1,25 @@
+package com.github.eyefloaters.console.api.model;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+public record ReplicaInfo(
+        @Schema(readOnly = true, description = "The total size of the log segments in this replica in bytes.")
+        long size,
```

Comment:
We need to defined whether this is node-local storage, or total storage, including remote log segments in an object store.

Reply:
This is a bit new to me. Is that information we can get from the `Admin` interface? From the limited testing I've done with it this is node-local storage, excluding the index storage.

Reply:
This is a new feature known as Tiered Storage, originally described in [KIP-405](https://cwiki.apache.org/confluence/display/KAFKA/KIP-405%3A+Kafka+Tiered+Storage). Luke Chen (@showuon) would be a good person to speak to if you want more info.

For our purposes here I _think_ it's enough to adjust the docs and maybe call this property `localSize`.

Reply:
This record was renamed to `ReplicaLocalStorage` with the `size` element kept as it was, but with a slight clarification to the description in the annotation.